### ISSUE TITLE
Wrong env stored in `io.sails.environment`

### ISSUE THREAD

**Issue by askmike** [balderdashy/sails.io.js#36]
Sails supports custom environments and will set `sails.config.environment` accordingly. However this lib will [try to determine](https://github.com/balderdashy/sails.io.js/blob/3ba4a87becdf19ebeb3a5331d53275cbb285e8ee/sails.io.js#L937) the environment (either `development` or `production`) based on whether the script src contains `min` or `production`. We do need the correct env in our frontend code.

---

Our current workaround is a custom API call.

**Comment by sailsbot** [comment]
Thanks for posting, @askmike.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by askmike** [comment]
This hasn't been fixed yet.
